crates/foundry/evm/core/src/backend/mod.rs

```diff
@@ -2249,12 +2249,6 @@ pub(crate) fn merge_account_data<ExtDB: DatabaseRef>(
         );
     }
 
-    // need to mock empty journal entries in case the current checkpoint is higher
-    // than the existing journal entries
-    while active_journaled_state.journal.len() > target_fork.journaled_state.journal.len() {
-        target_fork.journaled_state.journal.push(Vec::default());
-    }
-
```

Comment:
Need to look into this more. These lines are not present in Foundry but I don't know if that's because of some other change.

Reply:
This was removed along with the upgrade to revm v23. I didn't find any other change that would've justified this removal.